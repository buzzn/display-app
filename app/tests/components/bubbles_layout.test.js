import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import BubblesLayout from '../../components/bubbles_layout';

describe('bubbles layout component tests', () => {
  const renderer = new ShallowRenderer();

  test('should render proper styles', () => {
    renderer.render(<BubblesLayout Bubbles={ () => <div/> } registers={{}} Chart={ () => <div/> }/>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
