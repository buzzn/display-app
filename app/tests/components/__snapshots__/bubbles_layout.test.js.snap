// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bubbles layout component tests should render proper styles 1`] = `
<div
  className="bubbles-wrapper"
>
  <div
    className="row"
  >
    <div
      style={
        Object {
          "margin": "44px 156px",
          "position": "relative",
        }
      }
    >
      <div
        style={
          Object {
            "display": "inline-block",
            "height": "648px",
            "position": "relative",
            "width": "648px",
          }
        }
      >
        <Bubbles
          registers={Object {}}
        />
      </div>
    </div>
    <div
      style={
        Object {
          "height": "150px",
          "margin": "30px auto",
          "width": "832px",
        }
      }
    >
      <Chart />
    </div>
  </div>
</div>
`;
